.github/workflows/generate-translation-files.yml

```diff
@@ -0,0 +1,83 @@
+# This workflow generates translation files from code and moves the generated
+# translation files to the openedx-translations repository via an auto-merged pull
+# request.
+
+name: Generate Translation Files
+
+on:
+  # This action is meant to be used from other actions.
+  # https://docs.github.com/en/actions/learn-github-actions/reusing-workflows
+  - workflow_call
+
+jobs:
+  python-translations:
+    runs-on: ubuntu-20.04
```

Comment:
`ubuntu-latest` will work well enough for this, and won't have to be changed in five years or whatever :)